### PR TITLE
Functionality and Interactivity completed

### PR DESCRIPTION
**Pull request by coderacheal**
### **The following features have been implemented**

- [ ] Add tasks
- [ ] Remove task
- [ ] Edit task
- [ ] To-Do list saved in local storage.

### CODE REVIEW

**Review by t-yanick — CHANGES_REQUESTED** [review]
Hi @coderacheal , 

### Good job so far!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  App functioning as expected.  Good job
:heavy_check_mark: Descriptive PR
:heavy_check_mark: All linters checks passed

There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
- [ ] Please kindly add the exact procedure and steps to follow to run your app locally from the cloning of the repository.  This is absent from your ReadMe file, but it is very important for anyone who wants to test your app.
- I am doing the request here because your ReadMe file does not display in Files Changed.

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- Giving more Descriptive branch names. When you give a name like functionality, it is vague because you'll have many functionalities. 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 1]
```diff
@@ -33,11 +33,14 @@
...
+- Webpack
 
 
 ### Key Features <a name="Mobile Version of Portfolio"></a>
 
 - **Dynamically add list of tasks**
+- **Check tasks as completed**
+- **Delete checked items**
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
```
Comment (FrederickMih) [inline README.md:46]: Capture

- As the previous reviewer mentioned, the procedures to set up your app locally are not sufficient. The app needs to be opened on a browser using webpack dev server(`npm start`) after installing app dependencies using `npm install`

File: src/modules/taskFunctions.js [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+  updateIndex = () => {
+    this.tasks.forEach((task, index) => {
+      task.index = index + 1;
+    });
+  };
+
+  delete = (index) => {
+    this.tasks = this.tasks.filter((task) => task.index !== Number(index) + 1);
+  };
```
Comment (FrederickMih) [inline src/modules/taskFunctions.js:47]: Capture

- [Optional] I think we were told to implement a function for deleting **a single task**. Each task should have a delete button attached to it as you can see on the original video requirements. Right now I can only delete a task using the Clear all button

**Review by FrederickMih — APPROVED** [review]
## Hi 🙋  @coderacheal , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- All changes to the ToDo list are correctly saved in the local storage 👍 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**